bundle_gems

```diff
@@ -156,4 +189,5 @@ end
 File.open(File.join(outdir, spec), 'w') do |f|
   f.write(new_spec_lines.join)
 end
+FileUtils.remove_dir @bundler_directory
```

Comment:
isn't this a duplication of line 141? @hennevogel

Reply:
Line 141 only runs if mode is cpio.

It's a super weird implementation of the default, I can refactor this.

Reply:
ah right overlooked this